### ISSUE TITLE
Import error are not detected in travis

### ISSUE THREAD

**Issue by ThomasFaivre** [secdev/scapy#827]
As seen in #671, there is an issue with the detection of import errors in Travis. Using the provided patch, errors effectively stops the test  but I didn't manage to fix the issue with pcapy.

[0001-travis-allow-detection-of-import-error.txt](https://github.com/secdev/scapy/files/1324013/0001-travis-allow-detection-of-import-error.txt)

I tried installing in using `apt-get install python-pcapy` and `pip install pcapy` but it does not seem to work.

Here is an example of a travis job that passed but have an import error:
https://travis-ci.org/secdev/scapy/jobs/278282358

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "scapy/all.py", line 16, in <module>
    from scapy.arch import *
  File "scapy/arch/__init__.py", line 54, in <module>
    from scapy.arch.pcapdnet import *
  File "scapy/arch/pcapdnet.py", line 355, in <module>
    import pcapy as pcap
ImportError: No module named pcapy
```

Any ideas?

**Comment by guedou** [comment]
I can reproduced it with a clean Ubuntu install:
```
git clone --depth 1 https://github.com/secdev/scapy
cd scapy/
SCAPY_USE_PCAPDNET=yes python -c "from scapy.all import *; print conf"
```

I can be solved by doing 
`sudo apt install python-libpcap python-dumbnet`

I still need to investigate why this issue occurs are the Travis log indicate that both Python modules are installed.

**Comment by guedou** [comment]
Fixed with #844
